perfkitbenchmarker/vm_util.py

```diff
@@ -249,7 +249,8 @@ def WrappedFunction(*args, **kwargs):
             raise e
           else:
             if log_errors:
-              logging.error('Got exception running %s: %s', f.__name__, e)
+              logging.error('Got exception running %s: %s', f.__name__, e,
```

Comment:
Can this be made optional ? something akin to setting a log level. The whole stack trace may not be of interest in certain expected cases.